spec/controllers/course/virtual_classrooms_controller_spec.rb

```diff
@@ -15,32 +14,65 @@
 
     before { sign_in(user) }
 
-    describe '#index' do
-      context 'when virtual classrooms component is disabled' do
-        before do
-          allow(controller).
-            to receive_message_chain('current_component_host.[]').and_return(nil)
+    describe 'For students' do
```

Comment:
`context 'when the user is a student'` , `context` is always used for branches in our tests.
We uses most of conventions here: http://betterspecs.org/#contexts